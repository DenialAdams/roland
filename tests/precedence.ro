proc main() {
   5 + 50 / 5;
   print("we need a way to print numbers");
}
