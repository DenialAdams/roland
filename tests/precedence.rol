proc main() {
   let x: i64 = 5 + 50 / 5;
   let y: i64 = (5 + 50) / 5;
   print_int(x);
   print_int(y);
}
