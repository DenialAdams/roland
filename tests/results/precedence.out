we need a way to print numbers
